lib/widgets/buildTasks.dart

```diff
@@ -175,7 +175,7 @@ class _TasksBuilderState extends State<TasksBuilder> {
               )
             : ListView(
                 controller: scrollController,
-                padding: const EdgeInsets.symmetric(vertical: 4, horizontal: 2),
+                padding: const EdgeInsets.only(top: 4, left: 2, right: 2, bottom: 80),
```

Comment:
make the size dynamic using MediaQuery

Reply:
Hi @Pavel401!

I've made changes in this PR to introduce dynamic changes using MediaQuery. Although I didn't open a separate issue prior to this PR (I really apologize for that. I assure you this will never happen again), I've ensured that the changes are responsive and adhere to best practices for different screen sizes.

Please let me know if there are any further modifications needed.

Thank you!